src/extension/features/accounts/last-reconciled-date/settings.js

```diff
@@ -1,8 +1,15 @@
 module.exports = {
   name: 'LastReconciledDate',
-  type: 'checkbox',
-  default: false,
+  type: 'select',
+  default: '0',
```

Comment:
This should probably be 'disabled', since the other values are not digits, but strings?

Reply:
make sense @StephenBrown2 , i made that change and pushed.

Reply:
is there a slack or discord for this project ?

Reply:
Yep, we've a link in the README

Reply:
@joshmadewell , how can I get an invite per the README? "If you have any concerns or questions don't hesitate to reach out to us in an issue and ask for an invite to our Slack so you can collaborate with the team."  excited to join the slack conversation.

Reply:
You should just be able to click the discord button on the readme here: https://github.com/toolkit-for-ynab/toolkit-for-ynab#readme

That should launch you into our discord server.